### ISSUE TITLE
[FEATURE REQUEST]TCP/IP over SSH

### ISSUE THREAD

**Issue by nybu** [fabianfiorotto/quick-query#49]
**DBMS I'm using:**  MySQL
**OS I'm using:**   Win7

Would it be possible to add **TCP/IP over SSH** as a connection method like there is one in MySQL Workbench for example?

![ssh](https://user-images.githubusercontent.com/16501736/31164778-d389be3c-a8e8-11e7-9b38-98e30d17f7b7.PNG)

**Comment by fabianfiorotto** [comment]
This depends on https://github.com/mysqljs/mysql/pull/1110 that is inactive since 2015. I'm thinking to switch to mysql2 but it would require a lot of effort.

**Comment by fabianfiorotto** [comment]
I forked mysqljs so I decided to merge that PR and implemented SSH support. It assumes that the server runs in port 22 on the same host than MySQL.

**Comment by nybu** [comment]
Hi! Thank you very much for the update. In my case the server is running on port 10024. Is there any way i can change the default port?

**Comment by fabianfiorotto** [comment]
There isn't a proper way yet. It's hardcoded https://github.com/fabianfiorotto/quick-query/blob/9fdccaa9fd2d47fe7d37260e86e3fb576b5ab977/lib/quick-query-connect-view.coffee#L143

**Comment by nybu** [comment]
Hi, I changed the hardcoded port and was able to connect. Unfortunatelythe connection closes as soon as i try to do anything. Eg if i run a select query, i get the results but directly after the connection closes:

![image](https://user-images.githubusercontent.com/16501736/49500091-ba8e2d80-f867-11e8-9dc5-15a096786ec3.png)

Here i tried to change "Pension" to "Pension123". The update didnt work. Also after that I tried to see the columns of the "hp_bewertungen" table but wasnt able to. I have to force a reconnect to be able to do anything again.
